src/DqtApi/Models/GetTeacherRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace DqtApi.Models
+{
+    public struct GetTeacherRequest
+    {
+        public string TRN { get; set; }
```

Comment:
Is TRN a number?
```suggestion
        public int TRN { get; set; }
```

Reply:
No - TRN is text in CRM, and this makes more sense.  It's like a phone number - albeit it's made of digits, it's not really a 'number'.